cmd/bee/cmd/cmd.go

```diff
@@ -243,6 +245,8 @@ func (c *command) setAllFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint64(optionNameBlockTime, 15, "chain block time")
 	cmd.Flags().String(optionNameSwapDeploymentGasPrice, "", "gas price in wei to use for deployment and funding")
 	cmd.Flags().Duration(optionWarmUpTime, time.Minute*10, "time to warmup the node before pull/push protocols can be kicked off.")
+	cmd.Flags().Bool(optionNameMainNet, true, "triggers connect to main net bootnodes.")
```

Comment:
The main net is set to true, but default network id is 10 on line 217, which should result with an error from parseNetworks, by default settings.